docs/metrics/bertscore.md

```diff
@@ -83,11 +111,16 @@ To showcase the value of BERTScore, let's consider the following candidate and r
 
     | Candidate Text | Reference Text |
     | --- | --- |
-    | The sun set behind the mountains, casting a warm, orange glow across the horizon. | As the mountains obscured the sun, a warm, orange glow painted the horizon. |
-    | She sipped her coffee and gazed out of the window, lost in thought on a rainy afternoon. | Lost in thought on a rainy afternoon, she sipped her coffee and stared out of the window. |
-    | The adventurous explorer trekked through the dense jungle, searching for hidden treasures. | In search of hidden treasures, the intrepid explorer ventured through the dense jungle. |
-    | Laughter echoed through the park as children played on the swings and slides. | Children's laughter filled the park as they enjoyed the swings and slides. |
-    | The old bookstore was filled with the scent of well-worn pages, a haven for book lovers. | A haven for book lovers, the old bookstore exuded the fragrance of well-read pages. |
+    | The sun set behind the mountains, casting a warm, orange glow across the horizon. | As the mountains obscured the
+    sun, a warm, orange glow painted the horizon. |
+    | She sipped her coffee and gazed out of the window, lost in thought on a rainy afternoon. | Lost in thought on a
+    rainy afternoon, she sipped her coffee and stared out of the window. |
+    | The adventurous explorer trekked through the dense jungle, searching for hidden treasures. | In search of hidden
+    treasures, the intrepid explorer ventured through the dense jungle. |
+    | Laughter echoed through the park as children played on the swings and slides. | Children's laughter filled the
+    park as they enjoyed the swings and slides. |
+    | The old bookstore was filled with the scent of well-worn pages, a haven for book lovers. | A haven for book lovers,
+    the old bookstore exuded the fragrance of well-read pages. |
```

Comment:
<img width="668" alt="image" src="https://github.com/kolenaIO/kolena/assets/31229051/031bf2d4-8664-4642-a2b9-23304c552c7a">

I think the format got a bit messed up here. Should I create a commit to your branch to fix it?

Reply:
Ah thanks for catching this @billbai0102.  Will push a commit shortly to fix this.